docs/getting-started/GettingStarted.md

```diff
@@ -0,0 +1,95 @@
+# Getting Started
+
+-   [Getting Started](#getting-started)
+    -   [Add Carbon to a Website](#add-carbon-to-a-website)
```

Comment:
I would use web page instead of a web site to be a little specific.

Reply:
I am not an expert but I went based off of [this](https://writingexplained.org/webpage-vs-website-difference)